lib/toggle.js

```diff
@@ -42,13 +41,15 @@ const Knob = styled.span`
   background-color: var(--colors-background);
   height: calc(var(--space-3) - 4px);
   width: calc(var(--space-3) - 4px);
-  box-shadow: 0 0 4px rgba(0,0, 0,0.1);
+  box-shadow: 0 0 4px rgba(0, 0, 0, 0.1);
 `;
 
-const Shuttle = (props) => (
+const nbsp = ' ';
```

Comment:
Maybe `blank` instead since this isn't a true `&nbsp`?

Reply:
oh hm that was supposed to be a real nbsp, i'll fix that